src/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -29,20 +30,20 @@ public struct Complex : IEquatable<Complex>, IFormattable
         // This value is used inside Asin and Acos.
         private static readonly double s_log2 = Math.Log(2.0);
 
-        private double _real;
-        private double _imaginary;
+        private double m_real;
+        private double m_imaginary;
         
         public Complex(double real, double imaginary)
         {
-            _real = real;
-            _imaginary = imaginary;
+            m_real = real;
+            m_imaginary = imaginary;
```

Comment:
I would prefer we add comments.  I understand @sharwell's point, but at the same time, seeing a comment and avoiding a change entirely is a lot quicker than having to figure out why a test is failing, can be spotted in code reviews, etc.